spec/support/oauth_connections_helpers.rb

```diff
@@ -0,0 +1,30 @@
+module OAuthConnectionsHelpers
+  def mock_one_drive_authorization_validation(with: {})
```

Comment:
If someone has a better suggestion on how to name this method please let me know :slightly_smiling_face:

Reply:
eventually this will vanish for a VCR cassette, so I wouldn't bother too much. Name is fine.

Reply:
Actuall I tried using a VCR cassette first (`it "shows no option for manually managed permissions", vcr: "auth/one_drive/user_token" do`) but had some issues with it:
```
            An HTTP request has been made that VCR does not know how to handle:
              GET http://172.18.0.9:3000/__identify__
```
(The IP is from the docker container and probably needs to be ignored by VCR - I didn't dive into this).